### ISSUE TITLE
[Feature] Line chart view, toggle-able for certain dimensions

### ISSUE THREAD

**Issue by angelaambroz** [growthbook/growthbook#1833]
### Description of Feature

Our team would love the ability to view the treatment effects over time, indexing by the number of days since exposure. For example, the x-axis would be 1, 2, 3, etc. for how many days it's been since a user was triggered into the experiment. The y-axis would show either the absolute values of each bucket, or the relative lifts for each treatment bucket comparing against control.

This is a slight change from the existing line chart - the `Date Cohorts (First Exposure)` option - which shows us the total effects, by users grouped on their trigger date. 

### Related Issues

None.

CC: @itsgrimetime fyi

**Comment by lukesonnet** [comment]
This is definitely on our radar.

> This is a slight change from the existing line chart

You are right, but it requires up to roll up user results for each day  as they go through the experiment, which necessitates a more expensive query if you aren't just using sum metrics, so we may have to restrict it to those kinds of metrics.

**Comment by alyssamichelle** [comment]
Triage from the open-issues audit:

**Suggested action:** Likely close — verify (looks implemented)

**Why:** Time series charts exist; this asked for a toggleable line chart for certain dimensions. Please confirm in the product for a couple of minutes; if it still isn't done, say what's missing.

- [Time series](https://docs.growthbook.io/app/experiment-time-series)
- [Release 3.6.0](https://github.com/growthbook/growthbook/releases/tag/v3.6.0)

**Comment by angelaambroz** [comment]
Yep - this is implemented! I've moved companies and am no longer using Growthbook, but iirc this had been implemented and was in use. @itsgrimetime can confirm.
